docs/fidesops/docs/guides/reporting.md

```diff
@@ -77,60 +78,64 @@ Check out the [API docs here](/fidesops/api#operations-Privacy_Requests-get_requ
 
 ## View Individual Privacy Request Log Details
 
-Use the `verbose` query param to see more details about individual queries run as part of the Privacy Request along
-with individual statuses. 
+Use the `verbose` query param to see more details about individual collections visited as part of the Privacy Request along
+with individual statuses. Individual collection statuses include `in_processing`, `retrying`, `complete` or `error`.
```

Comment:
Should we add `paused` here too?

Reply:
Paused isn't a status for an ExecutionLog, it's just a status for the PrivacyRequest.

Reply:
Oh Of course. If a request is paused it's not executing anything. Gotcha